django_project/core/tests/test_check_domain_middleware.py

```diff
@@ -54,6 +54,6 @@ def test_invalid_domain_prod(self):
         self.assertEqual(response.status_code, 302)
         self.assertRedirects(
             response,
-            expected_url='http://changelog.qgis.org/en/domain-not-found/',
+            expected_url='http://changelog.kartoza.com/en/domain-not-found/',
```

Comment:
Same as above, Could you do a follow up commit that takes this out of settings rather? I don't think it is good to hard code URLs into the code.